src/gtk/toga_gtk/widgets/detailedlist.py

```diff
@@ -0,0 +1,55 @@
+from gi.repository import Gtk
+
+from .base import Widget
+
+class DetailedList(Widget):
+	def create(self):
+		self.native = Gtk.ScrolledWindow()
+		self.native.interface = self.interface
+
+		#Craete DetailedList
+		self.vbox = Gtk.VBox()
+
+		self.native.add_with_viewport(self.vbox)
+
+
+	def change_source(self, source):
+		# TODO
+		raise NotImplementedError()
```

Comment:
There's a 'not_implemented()' method in the factory that should be used in preference to this; the method version allows for a "soft" failure. The exception should only be raised if there is literally no way to recover, and there's no way to render the widget until a particular method is implemented - in which case, the patch probably isn't ready for merging.

Reply:
Okay; I'll update this with the soft logging option - should have done that in the first place, but didn't occur to me probably because **danyeaw** and I were already in discussions of doing the implementation together and will probably have it up in a few days :)